ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html

```diff
@@ -28,7 +28,7 @@
   data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
   data-placeholder-text="${wcmmode.edit && 'Please drag Vertical Tabs components here' @ i18n}"
   data-sly-test.widgetId="${'{0}-{1}' @ format=[panel.id, 'widget']}">
-     <div class="cmp-verticaltabs__label-container">
+     <div class="cmp-verticaltabs__label-container" role="heading">
```

Comment:
If you are adding the `role="heading"` attribute, you must also include the `aria-level="1"` to aria-level="6" attribute, depending on the appropriate heading level required. In this scenario, we should add `aria-level="2"`.